src/Symfony/Component/CssSelector/Node/SpecificityAdjustmentNode.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\CssSelector\Node;
+
+/**
+ * Represents a "<selector>:where(<subSelectorList>)" node.
+ *
+ * This component is a port of the Python cssselect library,
+ * which is copyright Ian Bicking, @see https://github.com/SimonSapin/cssselect.
+ *
+ * @author Hubert Lenoir <lenoir.hubert@gmail.com>
+ *
+ * @internal
+ */
+class SpecificityAdjustmentNode extends AbstractNode
+{
+    /**
+     * @param array<NodeInterface> $arguments
+     */
+    public function __construct(
+        public readonly NodeInterface $selector,
+        public readonly array $arguments = []
+    ) {
+    }
+
+    public function getSpecificity(): Specificity
+    {
+        return new Specificity(0, 0, 0);
```

Comment:
Does this class represent `<selector>:where(<subSelectorList>)` or just `:where(<subSelectorList>)` ? The specificity is 0 only in the second case

Reply:
This class includes the selector. I made changes to add selector specificity.

Reply:
Btw, this looks broken in the Python library as well.